terraform/examples/only_daos_server/terraform.tfvars.perf.example

```diff
@@ -0,0 +1,22 @@
+project_id                 = "<project_id>"
+region                     = "us-central1"
+zone                       = "us-central1-f"
+network                    = "default"
+subnetwork_project         = "<project_id>"
+subnetwork                 = "default"
+server_number_of_instances = 2
+server_labels              = {
+  example = "daos_cluster"
+}
+server_preemptible         = "false"
+server_mig_name            = "daos-server"
+server_template_name       = "daos-server"
+server_instance_base_name  = "daos-server"
+server_machine_type        = "n2-custom-36-262144"
+server_os_family           = "daos-server-centos-7"
+server_os_project          = "<project_id>"
+server_os_disk_type        = "pd-ssd"
+server_os_disk_size_gb     = 20
```

Comment:
Are these examples inverted? this looks like the TCO setup whereas the other one looks like perf.

Reply:
We fixed this on the call today.